lib/models/news.dart

```diff
@@ -12,8 +17,35 @@ class News {
     required this.audio,
     required this.paragraphs,
   });
+
+  factory News.fromJson(dynamic json) {
+    return News(
+        title: json['title'] as String,
+        date: json['date'] as String,
+        transcriptId: json['transcriptID'] as int,
+        audio: json['audio'] as String?,
+        paragraphs: (json['paragraphs'] != null)
+            ? (json["paragraphs"]) as List<Paragraph>
+            : List.empty());
+  }
```

Comment:
```suggestion
  factory News.fromJson(dynamic json) => _$NewsFromJson(json);
```

Can be done using the generated function.